app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+
+import { Table as NeetoUITable } from "neetoui";
+
+import Dropdown from "./Dropdown";
+import NameAndRole from "./NameRoleAndAvatar";
+import { getContactsData } from "./utils";
+
+import { noop } from "../utils";
```

Comment:
Move this util function to a global `utils.js` which we can access directly using an alias like `import { noop } from "utils";`

Reply:
Fixed with #32